### ISSUE TITLE
UNk Token 

### ISSUE THREAD

**Issue by samarIbrahem** [VT-vision-lab/VQA_LSTM_CNN#33]
i am wondering why we use unk token in test set ,will it affects the results ,and also if i hava a validation set used for early stopping .The encoded question must use the training vocab or add validation new words to vocab?
